ui/.template-lintrc.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
     },
     'require-input-label': 'off',
   },
-  ignore: ['lib/story-md', 'tests/**'],
+  ignore: ['lib/story-md', 'tests/**', 'app/templates/components/console/*'],
```

Comment:
I wonder if it would be better to use a block quote to disable the specific rule in the file? This disables linting for the entire file which then wouldn't catch any other errors.